src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -87,7 +82,11 @@ const SearchResultsPage = (props, context) => {
               aria-describedby="results-description"
             >
               {
-                !!(totalHits && totalHits !== 0) && (
+                !!(totalResults && totalResults !== 0) &&
+                  (<ResultsCount spinning={spinning} count={totalResults} />)
+              }
+              {
+                !!(totalResults && totalResults !== 0) && (
```

Comment:
If there are no results, then it shouldn't appear because there's nothing to sort.